src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,34 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Represents a command to find all tasks containing a keyword.
+ */
+public class FindCommand extends Command {
+    /** The keyword to find tasks with. */
+    private String keyword;
+
+    /**
+     * Creates a new find command with the specified keyword.
+     * @param keyword The keyword to find tasks with.
+     */
```

Comment:
This comment is missing the empty line between description and parameter section as stated in the coding convention page. Happens a few times in this class, so I'll just comment once here!

Reply:
Noted!